pkg/environmet/env.go

```diff
@@ -96,7 +96,7 @@ func (e *Env) Parse() error {
 		return fmt.Errorf("environment vars %v are missing", ls)
 	}
 	switch e.Mode {
-	case "oss", "pro":
+	case "ce", "pro":
```

Comment:
Are you sure this should be `ce` rather than `oss`?

Reply:
I have refactored, we had two env vars that were refering to the same thing. Some places used `ce` some places used `oss`. I decided to consolidate and use `ce` everywhere.

Reply:
does that introduce a breaking change though? can we change it so that we can accept `ce` or `oss`? we also need a docs update:

https://github.com/TykTechnologies/tyk-operator/blob/master/docs/installation/installation.md#tyk-operator-configuration

Reply:
@asoorm 

The docs have been updated here  cf74582
Added backward compatibility here 9b31fba so this is no longer a breaking change